### ISSUE TITLE
Objects.requireNonNullElse() and Objects.requireNonNullElseGet() should be treated the same as Objects.requireNonNull()

### ISSUE THREAD

**Issue by xzel23** [spotbugs/spotbugs#4166]
#2965 fixed this for `Objects.requireNonNull()`, but `Objects.requireNonNullElse()` and `Objects.requireNonNullElseGet()` are still treated as possibly returning `null`.

```java
public class Bug {
  public static void foo(CharSequence a) {
   int bar =  Objects.requireNonNullElse(a, "").length()
   int car =  Objects.requireNonNullElseGet(a, () -> "" + a).length()
  }
}
```

**Expected behavior**
There should be no warning reported.

**Actual behavior**
A warning is reported.

**Comment by xzel23** [comment]
My bad. I overlooked the same value being used again in the same line of code. Thanks for maintaining Spotbugs!
